### ISSUE TITLE
Bug: WinUI-2.5: Accessibility: [MAS2.4.3]: Keyboard: The loss of keyboard focus and improper focus order is observed while user navigate to the Pager control data grid using Tab key

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#3632]
**Tags**
Accessibility, A11yFT,  A11yMAS, A11yMediumImpact, MAS2.4.3, Keyboard, Sev3, WinUI-2.5, Win32, Wipro 

**User Impact**
Loss of keyboard focus or improper focus order may lead difficulty to the keyboard user. Thus it impacts the keyboard user.

**Repro-Steps**
1. Launch the WinUI_2.5_TestApp
2. Navigate to the Pager control tab
3. Navigate to the Pager control data grid and  using Tab key from Pager control tab element and observe the loss of keyboard focus and focus order in the data grid.
4. Repeat the same thing using Shift +Tab key from page control buttons.

**Expected Behavior**
No loss of keyboard focus should be observed while user navigate to the Pager control data grid using Tab key or shift + Tab keys.
And also focus order should be proper while user traverses the data grid elements using Tab or shift + Tab keys.

**Actual Behavior**
The loss of keyboard focus and improper focus order is observed while user navigate to the Pager control data grid using Tab key.
Using Shift + Tab keys also the keyboard focus loss and improper focus order is observed while user navigate to the Pager control data grid

**Repro attachments**

[Keybaord-Loss of Kb focus and improper order_edit.zip](https://github.com/microsoft/microsoft-ui-xaml/files/5558353/Keybaord-Loss.of.Kb.focus.and.improper.order_edit.zip)

**Hardware and build details**
Win32
Windows OS Version: Windows 10
App: WinUI_2.5_TestApp

**Comment by YuliKl** [comment]
Closing this issue because the data grid is not a WinUI control.

@MarissaMatt FYI in case you decide to use the data grid in the future. I recommend filing a bug on the owners of that control.
